frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -341,6 +341,27 @@ const HostDetailsPage = ({
     });
   }, [usersSearchString, host?.users]);
 
+  // Updates title that shows up on browser tabs
+  useEffect(() => {
+    const hostTab = () => {
+      if (location.pathname.includes("software")) {
+        return "software";
+      }
+      if (location.pathname.includes("schedule")) {
+        return "schedule";
+      }
+      if (location.pathname.includes("policies")) {
+        return "policies";
+      }
+      return "";
+    };
+
+    // e.g., Rachel's Macbook Pro schedule details | Fleet for osquery
+    document.title = `Host ${hostTab()} details | ${
+      host?.display_name || "Unknown host"
```

Comment:
Can we do something like this instead of `|| "Unknown host"`? Seems a little smoother when the host loads. This also applies to the other places where specific details conditionally populate the title, like Software:
```
const hostNameTitle = host?.display_name ? `| ${host.display_name} ` : "";
// e.g., Rachel's Macbook Pro schedule details | Fleet for osquery
document.title = `Host ${hostTab()} details ${hostNameTitle}| Fleet for osquery`;
```